modules/uncertainty/R/generate_joint_ensemble_design.R

```diff
@@ -57,35 +56,10 @@ generate_joint_ensemble_design <- function(settings,
     sampled_inputs[[input_tag]] <- input_result$ids
     design_list[[input_tag]] <- input_result$ids
   }
-  
-  # Sample parameters if we don't have it.
-  if (is.null(ensemble_samples)) {
```

Comment:
It needs to be detected a different way, but this function is still the one responsible for running get.parameter.samples if there's not already a samples.Rdata